.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Request creation of a new format specification'
+labels: spec-request
+---
+<!--
+A request for a format specification consists of 2 YAML blocks followed by a free-form text description.
+Fill in all the fields possible. Delete fields you cannot populate and all the comments from this template (including this one!).
+The resulting YAML blocks should contain valid YAML.
+Mandatory fields are marked with the word MANDATORY for your convenience.
+-->
+```yaml
+# This block is the boilerplate of a KSY spec file. It must be a valid KSY. It is meant to be ready to be copied as it is in order to make the task of the implementer easier.
```

Comment:
Is it probably the only comment I would have left, moved outside of YAML area and rewritten to left the user know why it is important, not just blindly throwing facts and "musts" and "mandatory" in one's face.